pvcompare/era5.py

```diff
@@ -118,15 +134,14 @@ def get_era5_data_from_datespan_and_position(
 
 
 def format_pvcompare(ds):
-    """
-    Format dataset to dataframe as required by the pvlib's ModelChain.
+    r"""
+    Format weather data to dataframe as required by the pvlib's ModelChain.
 
-    The pvlib's ModelChain requires a weather DataFrame with time series for
-    - wind speed `wind_speed` in m/s,
-    - temperature `temp_air` in C,
-    - direct irradiation 'dni' in W/m² (calculated later),
-    - global horizontal irradiation 'ghi' in W/m²,
-    - diffuse horizontal irradiation 'dhi' in W/m²
+    The `pvlib's <https://pvlib-python.readthedocs.io/en/stable/>`_
+    :py:func:`~.ModelChain` functionality requires a weather
```

Comment:
```suggestion
    ModelChain functionality requires a weather
```
unfortunately links like that do not work to other packages